lib/main.coffee

```diff
@@ -296,36 +288,49 @@ module.exports = Hydrogen =
             @moveDown endRow
 
     removeStatusBarElement: ->
-        if @statusBarElement?
-            while @statusBarElement.hasChildNodes()
-                @statusBarElement.removeChild(@statusBarElement.lastChild)
-
-    setStatusBarElement: (element) ->
-        if @statusBarElement?
-            @removeStatusBarElement()
-            @statusBarElement.appendChild(element)
-        else
-            console.error "No status bar element. Can't set it."
+        unless @statusBarElement?
+            console.error 'removeStatusBarElement: there is no status bar'
+            return
+
+        while @statusBarElement.hasChildNodes()
+            @statusBarElement.removeChild @statusBarElement.lastChild
+
+    setStatusBarElement: ->
+        unless @statusBarElement?
+            console.error 'setStatusBarElement: there is no status bar'
```

Comment:
If I restart atom and then start Hydrogen, this is the output:
<img width="853" alt="bildschirmfoto 2016-07-05 um 09 04 46" src="https://cloud.githubusercontent.com/assets/13285808/16576483/cc95728a-428f-11e6-86b3-4428778d0d67.png">